### DISCUSSION TITLE
我自己转换出来的推理模型与官方下载的不一样是怎么回事？

### DISCUSSION THREAD

**Discussion by wq881103** [PaddlePaddle/PaddleOCR#13128] [Q&A] [answered]
从模型列表里下载的ppocrv3中文识别推理模型，可以用在umi-ocr软件里面。下载模型列表里面的ppocrv3中文识别训练模型，使用的paddleocr2.6版本，自己转换成推理模型，用在umi-ocr软件里面就会报错。我自己转换出来的模型用tools/infer/predict_system.py就可以正常识别，有谁知道这是哪里出问题了吗？

**Comment by UserWangZz** [comment]
又具体的报错信息吗

**Reply by wq881103** [reply]
已经解决，旧版软件与新版模型不兼容的问题。把电脑上默认安装的paddlepaddle-gpu_2.6.1换成paddlepaddle-gpu_2.4.2.post117版本，就可以使用了

**Comment by SWHL** [comment]
你在umi-ocr软件里报错？这个建议去umi-ocr里问问，估计是它们哪里加载问题

**Reply by wq881103** [reply, marked as answer]
已经解决，旧版软件与新版模型不兼容的问题。把电脑上默认安装的paddlepaddle-gpu_2.6.1换成paddlepaddle-gpu_2.4.2.post117版本，就可以使用了
